tests/test_column_family.rs

```diff
@@ -143,15 +180,15 @@ fn test_merge_operator() {
             Err(e) => panic!("failed to open db with column family: {}", e),
         };
         let cf1 = db.cf_handle("cf1").unwrap();
-        assert!(db.put_cf(cf1, b"k1", b"v1").is_ok());
-        assert_eq!(db.get_cf(cf1, b"k1").unwrap().unwrap(), b"v1");
-        let p = db.put_cf(cf1, b"k1", b"a");
+        assert!(db.put_cf(&cf1, b"k1", b"v1").is_ok());
```

Comment:
Should we pass `cf1` by reference here ??? As I can see the signature of DB::cf_handle hasn't changed. And that's good !

Reply:
Good catch, reverted all the pass by references!